newIDE/app/src/EventsFunctionsList/EventsBasedObjectTreeViewItemContent.js

```diff
@@ -193,7 +206,21 @@ export class EventsBasedObjectTreeViewItemContent
   }
 
   renderRightComponent(i18n: I18nType): ?React.Node {
-    return null;
+    return this.eventsBasedObject.isPrivate() ? (
+      <Tooltip
+        title={
+          <Trans>This object won't be visible in the events editor.</Trans>
```

Comment:
```suggestion
          <Trans>This object won't be visible in the scene editor.</Trans>
```

Reply:
ok, I have too do the same change for behaviors.